src/2.2-counter-task/Counter.test.jsx

```diff
@@ -50,7 +74,27 @@ test.todo("allows increment and decrement");
  * 💡 Tips:
  * - To check if button is disabled use `toBeDisabled()` matcher
  */
-// test.todo("does not allow to go below min and above max");
+test("does not allow to go below min and above max", () => {
+    render(<Counter min={0} max={5} />);
+
+    const dec = screen.getByRole('button', { name: /decrement/i });
+    const inc = screen.getByRole('button', { name: /increment/i });
+
+    //Check if disabled.
+    expect(dec).toBeDisabled();
+
+    //Try to decrement.
+    runTimes(5, () => userEvent.click(dec))
```

Comment:
If button is disabled, click is automatically not allowed, so it might be redundant to test it here